Makefile

```diff
@@ -19,7 +19,6 @@ acceptance_test_deps=vendor-bin/behat/vendor
 PHPUNIT=php -d zend.enable_gc=0 ../../lib/composer/bin/phpunit
 PHPUNITDBG=phpdbg -qrr -d memory_limit=4096M -d zend.enable_gc=0 "../../lib/composer/bin/phpunit"
 PHP_CS_FIXER=php -d zend.enable_gc=0 vendor-bin/owncloud-codestyle/vendor/bin/php-cs-fixer
-PHP_CODESNIFFER=vendor-bin/php_codesniffer/vendor/bin/phpcs
```

Comment:
codesniffer is not used in this app, because there are no acceptance tests, and we only use codesnifffer for acceptance test code.